modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,120 @@
+= Modify API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirements change, you might want to modify an API Group by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. 
+
+You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service. However, modifying API Groups and API Group instances involves specific <<restrictions,restrictions>>.
+
+When you publish an API Group in Exchange, the <<exchange_status,status>> of the API Group changes based on specific scenarios. 
+
+[[restrictions]]
+== Restrictions for Modifying API Group and Instances
+
+Before you modify an API Group, ensure that you carefully review the restrictions:
+
+* Add new instances of APIs that already exist in the group only if the API Group instance has no contracts. 
+* Add only API instances that belong to the same identity provider (IdP). 
++
+You cannot mix IdPs in an API Group instance.
+* Add API instances but not APIs if:
+** The API Group instance is published.
+** The API Group version is published.
+** The API Group version belongs to more than one API Group instance.
+
+[[exchange_status]]
+== Exchange Status Based on API Group Changes
+
+The Exchange status of an API Group depends on these conditions:
+
+* If your API Group is published in Exchange, your Exchange asset displays the new API instance. 
+* If the API instance you added is related to an asset that is already linked to the Exchange API Groups asset, the API instance displays these details.
+* If the API instance you added belongs to a new asset that is not yet liked to the Exchange API Groups asset, customers must create a new version of the API Groups asset.
+
+//Also, this section is not about "changes" but specifically adding a new API instance, so maybe the title needs tweaking.--Kim
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
++
+Otherwise, you will not be able to find the required organization or business group in the list.
+* API Group Administrator permissions to add API instances to an API Group.
++
+The API Group Administrator permission is set at the business group level. Therefore, an API Group Administrator can create, update, delete all API Groups in the business group. The permissions also gives `View` access to all API instances of the business group to be able to add API instances to the API Group.
```

Comment:
```suggestion
The API Group Administrator permission is set at the business group level. Therefore, an API Group Administrator can create, update, and delete all API Groups in the business group. The permission also gives `View` access to all API instances of the business group to be able to add API instances to the API Group.
```

Reply:
Deleted this from this page. Done! Thanks!